Template/Components/Frame.html

```diff
@@ -10,6 +10,8 @@
                 <div>
                     <p>[* Διακονία */* Ministries *]</p>
                     <div>
+                        <a href="[*localization root*][* The%20Word.html */* The%20Word.html *]">[* The Word */* The Word *]</a>
```

Comment:
If these English names will be used even in Greek, you will want to [mark](https://www.w3schools.com/tags/ref_language_codes.asp) that they are actually English, not Greek. It will help screen readers pronounce it right, machine translators will produce slightly less‐bad translations when the outgoing language is tagged correctly, fonts can do language‐specific things properly, etc.

```suggestion
                        <a href="[*localization root*][* The%20Word.html */* The%20Word.html *]">[* <span lang="en-GB">The Word</span> */* The Word *]</a>
```